contrib/win32/win32compat/fileio.c

```diff
@@ -638,22 +639,53 @@ int
 fileio_stat(const char *path, struct _stat64 *buf)
 {
 	wchar_t* wpath = NULL;
-	int r = -1;
-
-	if ((wpath = utf8_to_utf16(path)) == NULL)
-		fatal("failed to covert input arguments");
-
-	r = _wstat64(wpath, buf);
-
-	/*
-	* If we doesn't have sufficient permissions then _wstat64() is returning "file not found"
```

Comment:
No, with GetFileAttributesExW, we don't need to have that permission. GetFileAttributesExW success does not mean you can call _wstat64.